WrightTools/tuning/spectral_delay_correction.py

```diff
@@ -51,15 +70,21 @@ def process_wigner(data_filepath, channel, control_name,
     else:
         print('channel type not recognized')
         return
-    # process
+     # clip slice
+    values = data.channels[channel_index].values
+    cutoffs = np.amax(values, axis=0)*slice_cutoff_factor  # caught by nans
+    print(cutoffs, cutoffs.shape)
+    values[values<cutoffs] = np.nan
+    data.channels[channel_index].values = values
+    # clip global
+    data.channels[channel_index].clip(min=data.channels[channel_index].max()*global_cutoff_factor)
+   # process
     function = wt_fit.Gaussian()
     fitter = wt_fit.Fitter(function, data, data.axes[0].name)
-    outs = fitter.run(channel_index)
+    outs = fitter.run(channel_index, propagate_other_channels=False, verbose=False)
     # clean
     # remove the edges because they are badly behaved...
     # should probably do something more sophisticated...
-    outs.amplitude.clip(outs.amplitude.max() * 0.1, outs.amplitude.max())
-    #outs.amplitude.clip(outs.amplitude.max()*0.04, outs.amplitude.max())
     outs.amplitude.values[0] = np.nan
     outs.amplitude.values[-1] = np.nan
     outs.width.clip(0, 500)
```

Comment:
This line and the one that clips based on the mean cause problems on the picosecond system (See #218). Now would probably be a good time to evaluate their utility. The ability to specify units as a parameter may help, but I'm not sure, would have to test.

Reply:
That is referring to `outs.width.clip(0,500)`